lib/mongo/server/app_metadata/platform.rb

```diff
@@ -34,21 +34,42 @@ def initialize(metadata)
           @metadata = metadata
         end
 
+        # Queries whether the current runtime is standard Ruby or not.
+        #
+        # @return [ Boolean ] whether the current runtime is standard Ruby
+        def ruby?
```

Comment:
"mri" ("matz's ruby implementation") is how the canonical implementation is usually called. It might be better to name this predicate `mri?` to reflect that, since `ruby?` is probably overly broad.

```suggestion
        def mri?
```

Reply:
fixed!